src/helpers/script_helpers.sh

```diff
@@ -191,6 +191,41 @@ function skippable {
 }
 
 
+# Adds a step to the current command
+function step {
```

Comment:
By itself I would agree with you. Given that it is called in a method called `steps` that defines the steps for a command, I think its okay and nicely short. I don't feel strongly about this though, so am happy to adjust.